odbc_fdw.c

```diff
@@ -174,7 +177,62 @@ static struct odbcFdwOption valid_options[] =
 	{ NULL,       InvalidOid}
 };
 
-typedef enum { TEXT_CONVERSION, HEX_CONVERSION, BIN_CONVERSION, BOOL_CONVERSION } ColumnConversion;
+typedef enum { TEXT_CONVERSION, BIN_CONVERSION, BOOL_CONVERSION } ColumnConversion;
+
+static GetDataTruncation
+result_truncation(SQLRETURN ret, SQLHSTMT stmt)
+{
+    SQLCHAR sqlstate[5];
+    GetDataTruncation truncation = NO_TRUNCATION;
+    if (ret == SQL_SUCCESS_WITH_INFO)
+    {
```

Comment:
The different styles (tabs and spaces) makes this harder to understand, specially when Github doesn't use the same 1 tab == x spaces as your editor. I would suggest one or the other and using something like clang-format to enforce it in your commits.

Reply:
I'm sorry I was messy with that, I'll try to fix my changes using tabs (I think that's we've been using mostly here) and I'll leave fixes in the resto of the file for a separate PR.